runtime/include/atomics/intrinsics/chpl-atomics.h

```diff
@@ -339,20 +339,6 @@ DECLARE_ATOMICS(uint_least16_t);
 DECLARE_ATOMICS(uint_least32_t);
 DECLARE_ATOMICS(uint_least64_t);
 
-// On netbsd the DECLARE_ATOMICS macro doesn't work for uintptr_t. From gbt:
```

Comment:
Except, now there's no explanation for why we don't just do `DECLARE_ATOMICS(uintptr_t)` here.  Maybe we don't need the level of detail that was there before, but it looks a little odd to do `uintptr_t` differently with no explanation.

Reply:
Yeah, good point. 

You ok with me just re-adding "On netbsd the DECLARE_ATOMICS macro doesn't work for uintptr_t" and leaving the larger explanation for `git log`?

Reply:
Yeah, I think that would be fine.